### PR TITLE
Added training parameter to batch_normalization

### PR DESCRIPTION
**Pull request by PiotrCzapla**
According to the docs the batch_normalization layer does not work properly if the parameter is not set correctly.

### CODE REVIEW

File: tensorflow/examples/learn/resnet.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def res_net_model(features, labels, mode):
         filters=64,
         kernel_size=7,
         activation=tf.nn.relu)
-    net = tf.layers.batch_normalization(net)
+    net = tf.layers.batch_normalization(net, training=mode == tf.estimator.ModeKeys.TRAIN)
```
Comment (rmlarsen) [inline tensorflow/examples/learn/resnet.py]: Just calculate a boolean 

```training = mode == tf.estimator.ModeKeys.TRAIN```

and pass that to batch_normalization. Same everywhere.

**Review by rmlarsen — CHANGES_REQUESTED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmlarsen** [comment]
@PiotrCzapla could you address my comments so we can merge this?

**Comment by PiotrCzapla** [comment]
@rmlarsen thanks for making me fix this style issues. It make sense to keep the examples clean.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by rmlarsen — APPROVED** [review]
